src/features/MenuPage/MenuCourses/MenuItem.js

```diff
@@ -1,7 +1,71 @@
 import 'materialize-css';
-// import { Button, Card, Row, Col } from 'react-materialize';
+import { Button} from 'react-materialize';
+import { useState} from "react";
+
 
 const MenuItem = (props) => {
+
+    const [menuItemId] = useState (props.menuItemId)
+    const [quantity, setQuantity] = useState(0)
+    const orderId = localStorage.getItem('orderId')
+
+    const addToQuantity = () => {
+        setQuantity(quantity + 1)
+    }
+
+    const removeFromQuantity = () => {
+                if (quantity !== 0 ) {
+                    setQuantity(quantity - 1)
+                }
+    }
+
+    const order = {
+        orderId: orderId,
+        orderItems: [
+            {
+                menuItemId: menuItemId,
+                quantity: quantity
+            }
+        ]
+    }
+
+    const addItemToOrder = (order) => {
+        console.log(order)
+        if (order.orderItems[0].quantity !== 0) {
```

Comment:
I'd suggest adding an else condition here that displays an error in case someone tries to add an item with 0 quantity